plugins/workbench-resources/src/components/Logo.svelte

```diff
@@ -13,23 +13,15 @@
 // limitations under the License.
 -->
 <script lang="ts">
-  import type { IntlString } from '@hcengineering/platform'
-  import { tooltip } from '@hcengineering/ui'
-  import Arrows from './icons/Arrows.svelte'
-
   export let bundle: 'platform' | 'ezthera' = 'platform'
-  export let label: IntlString
-  export let selected: boolean
 </script>
 
-<button class="antiLogo {bundle}" class:selected tabindex="0" use:tooltip={{ label }} on:click>
-  <span class="logo">{bundle === 'ezthera' ? 'E' : 'P'}</span>
-  <div class="arrows"><Arrows size={'small'} /></div>
-</button>
+<div class="antiLogo {bundle}">
+  {bundle === 'ezthera' ? 'E' : 'P'}
```

Comment:
Nope, Please leave as P and create issue to me or Denis to make it configurable.